docs/conf.py

```diff
@@ -1,5 +1,5 @@
 # pylint: disable=invalid-name
-# Requires Python 3.6+
+# Requires Python 3.7+
```

Comment:
This is not true. The code here does not use any features that are Python 3.7 specific. And it does not have exactly the same dependencies as the runtime. Sphinx is being executed in a separate env that mustn't be tightly coupled with the test/dev/runtime envs.